designcompose/src/main/java/com/android/designcompose/squoosh/SquooshTreeBuilder.kt

```diff
@@ -488,58 +479,30 @@ internal fun generateReplacementListChildNode(
     childIdx: Int,
     layoutIdAllocator: SquooshLayoutIdAllocator,
 ): SquooshResolvedNode {
-    val layoutStyle = LayoutStyle.newBuilder()
-    val nodeStyle = NodeStyle.newBuilder()
-
-    val itemStyle =
-        ViewStyle.Builder()
-            .apply {
-                layout_style = Optional.of(layoutStyle.build())
-                node_style = Optional.of(nodeStyle.build())
-            }
-            .build()
+    val listChildViewData = container { shape = viewShape { rect = box { isMask = false } } }
 
-    val listChildViewData =
-        Container.Builder()
-            .apply {
-                shape = Optional.of(newViewShapeRect(false))
-                children = emptyList()
-            }
-            .build()
+    val listChildScrollInfo = scrollInfo {
+        pagedScrolling = false
+        overflow = OverflowDirection.OVERFLOW_DIRECTION_NONE
+    }
 
-    val listChildScrollInfo =
-        ScrollInfo.Builder()
-            .apply {
-                paged_scrolling = false
-                overflow = OverflowDirection.None().toInt()
-            }
-            .build()
-
-    val listChildView = View.Builder()
-    listChildView.unique_id = 0 // This is unused.
-    listChildView.id = "replacement-${node.view.id}-${childIdx}"
-    listChildView.name = "Replacement List ${node.view.name} / $childIdx"
-    listChildView.component_info = Optional.empty()
```

Comment:
Not sure if it works without setting any value to the `componentInfo` which used to be Optional.empty()

Reply:
It works, new messages have all fields unset.